storage/storage.go

```diff
@@ -232,7 +232,6 @@ func NewClient(ctx context.Context, opts ...option.ClientOption) (*Client, error
 // You may configure the client by passing in options from the [google.golang.org/api/option]
 // package.
 func NewGRPCClient(ctx context.Context, opts ...option.ClientOption) (*Client, error) {
-	opts = append(defaultGRPCOptions(), opts...)
```

Comment:
What's your reasoning for removing them here vs in `newGRPCStorageClient`? Looking through the code, I'm 95% sure that it doesn't matter which of the two gets removed in terms of functionality, but curious to know if there was logic behind which we're keeping.

Reply:
I originally did that (and moved the gax option to the constructor) but tests started failing; I then realized that the emulator tests call `NewGRPCStorageClient` directly. We could also pass the options to the test initialization but it seemed simpler just to put everything here.